### PR TITLE
CHET-329: Add learn more link

### PR DESCRIPTION
**Pull request by nanux**
* added Learn More link that links to KB article with description for the downtime.
** The anchor doesn't work properly but I am not sure if this is expected and @ddomingorh does have a version prepared or we need to fix the link. I've left a comment on https://aws-partner.atlassian.net/browse/CHET-329?focusedCommentId=10144
* I've restyled the page so it is more spacious. I think it would be good to have some unified and consistent styles across the page but I would leave that post-MVP
* Lint fix

image

### CODE REVIEW

File: frontend/src/main/dc-migration-assistant-fe/components/shared/MigrationTransferPage.tsx [hunk 1]
```diff
@@ -19,12 +19,12 @@ import SectionMessage from '@atlaskit/section-message';
 import styled from 'styled-components';
 import moment from 'moment';
 import Spinner from '@atlaskit/spinner';
+import { I18n } from '@atlassian/wrm-react-i18n';
```
Comment (nanux) [inline frontend/src/main/dc-migration-assistant-fe/components/shared/MigrationTransferPage.tsx:22]: just lint

File: frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx [hunk 2]
```diff
@@ -8,23 +8,62 @@ import { dbPath } from '../../utils/RoutePaths';
...
 `;
 
 const nextButtonStyle = {
     padding: '5px',
     marginRight: '20px',
 };
 
+const LearnMore: FunctionComponent = () => {
+    const LearnMoreLink =
+        'https://confluence.atlassian.com/jirakb/how-to-use-the-data-center-migration-app-to-migrate-jira-to-an-aws-cluster-1005781495.html#HowtousetheDataCenterMigrationapptomigrateJiratoanAWScluster-downtimepage';
```
Comment (nanux) [inline frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx:56]: @ddomingorh, is this the correct link? I took it from your comment but the anchor doesn't wokr.

File: frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx [hunk 3]
```diff
@@ -8,23 +8,62 @@ import { dbPath } from '../../utils/RoutePaths';
...
+        <Paragraph>
+            <a target="_blank" rel="noreferrer noopener" href={LearnMoreLink}>
+                {I18n.getText('atlassian.migration.datacenter.common.learn_more')}
+            </a>
+        </Paragraph>
+    );
+};
+
 export const WarningStagePage: FunctionComponent = () => {
     const [agreed, setAgreed] = useState<boolean>(false);
```
Comment (theghostwhoforks) [inline frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx:68]: Do we need both the flags or can we do with just one? 

i.e. `confirmOffline` can replace both `agree` and `shouldRediect` correct?

File: frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx (hunk 2 above)
Comment (theghostwhoforks) [reply frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx:56]: I'm assuming this is the correct link 
`https://confluence.atlassian.com/jirakb/how-to-use-the-data-center-migration-app-to-migrate-jira-to-an-aws-cluster-1005781495.html#HowtousetheDataCenterMigrationapptomigrateJiratoanAWScluster-downtimepageMigrationschedule`

Comment (theghostwhoforks) [reply frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx:56]: But @ddomingorh should confirm this

File: frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx (hunk 3 above)
Comment (nanux) [reply frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx:67]: Simplified :)

File: frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx (hunk 2 above)
Comment (ddomingorh) [reply frontend/src/main/dc-migration-assistant-fe/components/warning/WarningStage.tsx:55]: Yep the link should work. Tested it now, copy-pasting it directly to the browser works.

**Pull request merged**